deploy/backplane/srep/fedramp/10-srep-fedramp-muo.Role.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: backplane-srep-fedramp-muo
+  namespace: openshift-managed-upgrade-operator
+rules:
+# allows SREP to create upgradeconfigs for upgrades
+# until MUO-OCM functionallity is working
+# https://issues.redhat.com/browse/XCMSTRAT-240
+- apiGroups:
+  - upgrade.managed.openshift.io
+  resources:
+  - upgradeconfigs
+  verbs:
+  - create
```

Comment:
Do we want to be able to edit/delete them too? Probably a good idea if we are manually managing them

Reply:
If I'm reading this right, we already have delete permissions granted [HERE](https://github.com/openshift/managed-cluster-config/blob/d6acfd5e1cdc4d79a96fbf40a663e5e992998023/deploy/acm-policies/50-GENERATED-backplane-srep.Policy.yaml#L324)

```
$ oc auth can-i delete upgradeconfig 
yes
```

As for edit, I dont know that I've had to edit before and I'm good with adding it if its ok. I was keeping things limited as asked and create/delete seemed enough

Reply:
Okay, that makes sense. Thanks!